.changeset/stupid-shoes-complain.md

```diff
@@ -0,0 +1,7 @@
+---
+'astro': minor
```

Comment:
This seems like a bug fix to me rather than a new feature. Am I wrong?

Reply:
This is changing the way we compute `slug`s based on today's documentation. I chose a minor since this is breaking!

Reply:
Ok, I think this is fine for now. We should probably discuss as a team how to handle semver for experimental features. In the beta period it will be equivalent to patches I believe.

Reply:
Experimental features should not be coverd by `semver` right? I read this from Next.js